clusterloader2/pkg/measurement/common/wait_for_pvcs.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"time"
+
+	"k8s.io/klog"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+const (
+	defaultWaitForPVCsTimeout       = 60 * time.Second
+	defaultWaitForPVCsInterval      = 5 * time.Second
+	waitForBoundPVCsMeasurementName = "WaitForBoundPVCs"
+)
+
+func init() {
+	if err := measurement.Register(waitForBoundPVCsMeasurementName, createWaitForBoundPVCsMeasurement); err != nil {
+		klog.Fatalf("Cannot register %s: %v", waitForBoundPVCsMeasurementName, err)
+	}
+}
+
+func createWaitForBoundPVCsMeasurement() measurement.Measurement {
+	return &waitForBoundPVCsMeasurement{}
+}
+
+type waitForBoundPVCsMeasurement struct{}
```

Comment:
@krzysied - we should do that for all existing measurements.

Reply:
All measurements register create function: func() -> Measurement. IMHO this should trigger type check during compilation.

Reply:
The create measurement func should do the type check because it returns that waitForBoundPVCsMeasurement struct as the desired Measurement interface.